src/shared/tpm2-util.c

```diff
@@ -1590,6 +1780,20 @@ int tpm2_seal(const char *device,
         if (!hash)
                 return log_oom();
 
+        /* serialize the key for storage in the LUKS header. A deserialized ESYS_TR provides both
+         * the raw TPM handle as well as the object name. The object name is used to verify that
+         * the key we use later is the key we expect to establish the session with.
+         */
+        if (ret_primary_buf) {
+                log_debug("Serializing SRK");
```

Comment:
this log message is a bit misleading, no? after all you cannot extract the SRK, and cannot serialize it. You only serialize the reference to it here. hence maybe say that